docs/guides/development-guides/how-to-show-and-hide-a-split-view-pane-with-mvvm.md

```diff
@@ -3,17 +3,304 @@ id: how-to-show-and-hide-a-split-view-pane-with-mvvm
+Once you created the project you should be presented with the following files:
+- MainWindow.axaml
+- MainWindowViewModel.cs
 
+Those are the files that you will be editing, except for the one that you will be creating during this guide.
+
+:::info
+Although here I will be using the MainWindow.axaml and MainWindowViewModel.cs files, this guide will work for every other View and its respective ViewModel.
+:::
+
+## Adding the SplitView panel to the MainWindow
+The starting point will be something like this:
+``` xml
+<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:vm="using:ciao.ViewModels"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+        x:Class="ciao.Views.MainWindow"
+        x:DataType="vm:MainWindowViewModel"
+        Icon="/Assets/avalonia-logo.ico"
+        Title="ciao">
+
+    <Design.DataContext>
+        <!-- This only sets the DataContext for the previewer in an IDE,
+             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
+        <vm:MainWindowViewModel/>
+    </Design.DataContext>
+
+    <TextBlock Text="{Binding Greeting}" HorizontalAlignment="Center" VerticalAlignment="Center"/>
+
+</Window>
+```
+
+For the sake of simplicity we will be deleting the default TextBlock and its value in the MainWindowViewModel.
+<br>
+Now, create the panel and define the basics of how you want it to behave (for more informations on how this control works see [here](https://docs.avaloniaui.net/docs/reference/controls/splitview)) 
+``` xml
+<SplitView PanePlacement="Right" DisplayMode="CompactInline">
+</SplitView>
+```
+If you used this same code you should see the following window when starting the application. The light grey panel on the right is the one you will be expanding at the end of this guide.
+<img className="screenshot-full" src={FirstExample} alt="First example" />
+
+## Adding the SplitView's pane
+Now that we have the SplitView we need to create its pane, which is the part that will be hidden when the split view is closed.
+``` xml
+<SplitView PanePlacement="Right" DisplayMode="CompactInline">
+    <SplitView.Pane>
+
+    </SplitView.Pane>
+</SplitView>
+```
+We will now add some additional elements to the pane. First a StackPanel, then a button and a textbox with the text "Settings". The following configurations will ensure that the button and the text are aligned.
+``` xml
+<SplitView PanePlacement="Right" DisplayMode="CompactInline">
+    <SplitView.Pane>
+        <StackPanel VerticalAlignment="Top" Margin="5" Orientation="Horizontal">
+            <Button>
```

Comment:
Yes, I know. I kept it open to avoid causing confusion to someone who has never seen it written like that. The Button gets used later when adding content to it.